pkg/helmer/helmer.go

```diff
@@ -163,6 +164,43 @@ func (h *helmer) logWrap(format string, v ...interface{}) {
 	h.log.Info("Helm", "internal", msg)
 }
 
+func (h *helmer) failRelease(rel *release.Release, err error) error {
```

Comment:
The [Helm code](https://github.com/helm/helm/blob/main/pkg/action/install.go#L435) also uninstalls the release. Don't we want to replicate that behaviour in SRO?

Reply:
This is only done when the release is configured to be atomic, which is either install everything, or nothing. Not our case as per https://github.com/openshift-psap/special-resource-operator/blob/master/pkg/helmer/helmer.go#L202-L211